compiler/ast.nim

```diff
@@ -269,7 +269,8 @@ type
                       # language; for interfacing with Objective C
     sfDiscardable,    # returned value may be discarded implicitly
     sfOverriden,      # proc is overriden
-    sfGenSym          # symbol is 'gensym'ed; do not add to symbol table
+    sfGenSym,         # symbol is 'gensym'ed; do not add to symbol table,
+    sfCustomPragma    # symbol is custom pragma template
```

Comment:
I would introduce an alias to an existing sfFlag here to keep the number of flags below 32.

Reply:
Changed to the alias of existing sfFlag. As alternative I can make a new SymKind, I know it is a bit of work, but if you find it to be better solution I can do that.

Reply:
No new SymKind. Why? Because the symbol kinds should reflect the concrete syntax, the syntax is ``template foo {.pragma.}`` and so it should be an skTemplate with an sfPragma flag.